Test failure. Not sure if related.
```
FAIL: testapps.test_elasticsearch.tests.test_prod_es_indices:ProdIndexManagementTest.test_prod_config
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/django/test/utils.py", line 384, in inner
    return func(*args, **kwargs)
  File "/mnt/commcare-hq-ro/testapps/test_elasticsearch/tests/test_prod_es_indices.py", line 31, in test_prod_config
    self.assertEqual(EXPECTED_PROD_INDICES, found_prod_indices)
AssertionError: Lists differ: [{u'alias': u'case_search', u'... != [{'alias': u'case_search', u'm...
First differing element 1:
{u'alias': u'hqapps', u'meta': {u'settings': {u'analysis': {u'analyzer': {u'default': {u'filter': [u'lowercase'], u'type': u'custom', u'tokenizer': u'whitespace'}}}}}, u'type': u'app', u'index': u'test_hqapps_2017-05-22_1426'}
{'alias': u'hqapps', u'meta': {u'number_of_replicas': 0, u'settings': {u'analysis': {u'analyzer': {u'default': {u'filter': [u'lowercase'], u'type': u'custom', u'tokenizer': u'whitespace'}}}}}, 'type': u'app', 'index': u'test_hqapps_2017-05-22_1426'}
Diff is 8972 characters long. Set self.maxDiff to None to see it.
```

--- a/env_settings.py
+++ b/env_settings.py
@@ -41,8 +41,8 @@ ES_META = {
     },
 
     'hqapps': {
-        "number_of_replicas": 0,
         "settings": {
+            "number_of_replicas": 0,
             "analysis": {
                 "analyzer": {
                     "default": {

--- a/testapps/test_elasticsearch/tests/test_prod_es_indices.py
+++ b/testapps/test_elasticsearch/tests/test_prod_es_indices.py
@@ -66,6 +66,7 @@ EXPECTED_PROD_INDICES = [
         "type": "app",
         "meta": {
             "settings": {
+                "number_of_replicas": 0,
                 "analysis": {
                     "analyzer": {
                         "default": {
@@ -111,6 +112,7 @@ EXPECTED_PROD_INDICES = [
         "type": "hqdomain",
         "meta": {
             "settings": {
+                "number_of_replicas": 0,
                 "analysis": {
                     "analyzer": {
                         "default": {

